appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1545,25 +1545,41 @@ definitions:
         allOf:
           - $ref: '#/definitions/Curies'
           - $ref: '#/definitions/SelfLink'
+      date:
+        type: string
+        example: 2016-W2
+        description: The week date in ISO 8601 format (YYYY-'W'w)
       _embedded:
         properties:
           "yona:weekActivities":
             type: array
             items:
               $ref: '#/definitions/WeekActivity'
+    required:
+      - date
   WeekActivity:
     allOf:
       - $ref: '#/definitions/IntervalActivity'
     properties:
       date:
         type: string
         example: 2016-W2
-        description: The week date in ISO 8601 format (YYYY-'W'w)
+        description: The week date in ISO 8601 format (YYYY-'W'w). Only included in week detail
```

Comment:
Why is this? Now the app needs to do math to find out that the "batch" starts in week 5 and this is the 3 item in the batch, so this is week 8.